### PR TITLE
Fixed documentation - removed section about redis

### LINKED ISSUE #1380: Doc issues: Installing a development environment
https://mxcubeweb.readthedocs.io/en/latest/dev/environment.html

This is not a thing (see https://github.com/conda/conda/issues/12313)

```
conda env create -f conda-environment.yml python=3.9

SpecNotFound: Invalid name 'python=3.9', try the format: user/package
```

I guess you can do this

```
conda create python=3.9 --name mxcubeweb
conda env update --name mxcubeweb --file conda-environment.yml
```

but it reverts to python 3.11 because

```
The following packages are incompatible
├─ nodejs 18.*|20.*  is installable with the potential options
│  ├─ nodejs [18.19.0|18.20.2|20.12.2|20.9.0] would require
│  │  └─ openssl >=3.2.1,<4.0a0 , which can be installed;
│  ├─ nodejs [18.20.3|20.12.2|...|22.5.1] would require
│  │  └─ openssl >=3.3.1,<4.0a0 , which can be installed;
│  ├─ nodejs 20.9.0 would require
│  │  └─ openssl >=3.1.4,<4.0a0 , which can be installed;
│  └─ nodejs [18.10.0|18.11.0|...|20.8.1], which can be installed;
├─ pnpm 9**  is installable with the potential options
│  ├─ pnpm [9.0.1|9.0.2|...|9.0.6] would require
│  │  └─ nodejs >=20.9.0,<21.0a0 , which can be installed (as previously explained);
│  ├─ pnpm [9.0.1|9.0.2|...|9.9.0] would require
│  │  └─ nodejs [>=18.19.0,<19.0a0 |>=18.20.2,<19.0a0 |>=20.12.2,<21.0a0 ], which can be installed (as previously explained);
│  └─ pnpm [9.4.0|9.5.0|...|9.9.0] would require
│     └─ nodejs [>=22.4.0,<23.0a0 |>=22.4.1,<23.0a0 |>=22.5.1,<23.0a0 |>=22.6.0,<23.0a0 ] with the potential options
│        ├─ nodejs [18.20.3|20.12.2|...|22.5.1], which can be installed (as previously explained);
│        └─ nodejs [22.6.0|22.7.0] would require
│           └─ openssl >=3.3.1,<4.0a0 , which can be installed;
└─ python 3.9  is not installable because it requires
   └─ openssl >=1.1.1h,<1.1.2a , which conflicts with any installable versions previously reported.
```

Missing package

```
conda install redis-server
```

The package `authlib` is not optional

```
Traceback (most recent call last):
  File "/home/denolf/dev/sb/mxcubeweb/mxcubeweb/__init__.py", line 123, in build_server_and_config
    mxcube.init(
  File "/home/denolf/dev/sb/mxcubeweb/mxcubeweb/app.py", line 296, in init
    _UserManagerCls = import_component(
                      ^^^^^^^^^^^^^^^^^
  File "/home/denolf/dev/sb/mxcubeweb/mxcubeweb/core/components/component_base.py", line 22, in import_component
    mod = importlib.import_module(_module)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/denolf/.local/wdnbashutils/virtualenvs/mxcubeweb/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/denolf/dev/sb/mxcubeweb/mxcubeweb/core/components/user/usermanager.py", line 12, in <module>
    from authlib.integrations.flask_client import OAuth
ModuleNotFoundError: No module named 'authlib'
```

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Closes #1380 

- Fixed section about `conda` using incorrect syntax
- Removed sections about redis-server

### CODE REVIEW

File: docs/source/dev/environment.md [hunk 1]
```diff
@@ -57,8 +57,9 @@ conda env create -f conda-environment.yml --name another_name
 To choose a specific Python version when creating the conda environment,
 for example Python 3.9, one can use a command such as the following:
 
-```none
-conda env create -f conda-environment.yml python=3.9
+```
+conda create python=3.9 --name mxcubeweb
+conda env update --name mxcubeweb --file conda-environment.yml
```
Comment (fabcor-maxiv) [inline docs/source/dev/environment.md]: The [issue ticket](https://github.com/mxcube/mxcubeweb/issues/1380#issue-2492532184) says that this technique does not work. Or did I misunderstand something?

File: docs/source/dev/environment.md [hunk 2]
```diff
@@ -85,11 +86,6 @@ pnpm --prefix ui build
 ### 7. Running the application (server)
 
 ```
-# Start redis-server (if not already running)
-# Open a new terminal:
-conda activate mxcubeweb
-redis-server
-
 # In the previous terminal:
```
Comment (fabcor-maxiv) [inline docs/source/dev/environment.md]: This line seems superfluous as well.

```suggestion
```

**Review by fabcor-maxiv — CHANGES_REQUESTED** [review]

File: docs/source/dev/environment.md (hunk 1 above)
Comment (marcus-oscarsson) [reply docs/source/dev/environment.md]: The above should work, however we cant pass `python=` to `conda env create` like `conda env create -f conda-environment.yml python=3.9` its not valid syntax.

Then there  might be some other dependency issue, but that is a separate problem

Comment (fabcor-maxiv) [reply docs/source/dev/environment.md]: OK then I suggest using Python 3.10 as an example. It seems like it does resolve. At least it does resolve with micromamba: `micromamba create -f conda-environment.yml 'python==3.10'`.

```suggestion
conda create python=3.10 --name mxcubeweb
conda env update --name mxcubeweb --file conda-environment.yml
```

Comment (fabcor-maxiv) [reply docs/source/dev/environment.md]: And then if we have an issue with conda not being able to resolve for Python 3.8 or Python 3.9, then we should address it. I do not see much point in fixing the documentation by writing something that we already know does not deliver the expected result.

That is why I always prefer we solve one issue at a time. This pull request's title is about Redis, so I'd recommend we handle only Redis here and look at the rest of the issues one after the other in separate issues/PRs.

Comment (marcus-oscarsson) [reply docs/source/dev/environment.md]: Ok

**Review by fabcor-maxiv — APPROVED** [review]

File: docs/source/dev/environment.md (hunk 1 above)
Comment (fabcor-maxiv) [reply docs/source/dev/environment.md]: https://github.com/mxcube/mxcubeweb/issues/1383

**Pull request merged**